ps_facetedsearch.php

```diff
@@ -543,24 +550,24 @@ public function indexProductPrices($idProduct, $smart = true)
                     $minPriceValue = array_key_exists($idCountry, $minPrice) ? $minPrice[$idCountry][$currency['id_currency']] : 0;
                     $maxPriceValue = array_key_exists($idCountry, $maxPrice) ? $maxPrice[$idCountry][$currency['id_currency']] : 0;
                     if (!in_array($taxRateByCountry['iso_code'], self::ISO_CODE_TAX_FREE)) {
-                        $minPriceValue = Tools::ps_round($minPriceValue * (100 + $taxRate) / 100, 0);
-                        $maxPriceValue = Tools::ps_round($maxPriceValue * (100 + $taxRate) / 100, 0);
+                        $minPriceValue = Tools::ps_round($minPriceValue * (100 + $taxRate) / 100, self::DECIMAL_DIGITS);
```

Comment:
So you go for 5 digits instead of 0 ?

Reply:
Yep because of rounding.
Example:
 - Red scarf 10.60€
 - Blue scarf 10.80€
 - Yellow scarf 10.70€

The previous behaviour was saving 11€ for each products, which means you're not able to properly request and check the price order :/
I use 5 because digits will be configurable in the BO, and can be higher than 2.